pallets/bridge/Cargo.toml

```diff
@@ -31,8 +32,8 @@ common-traits = { path = "../../libs/common-traits", default-features = false }
 
 [dev-dependencies]
 # Testing/mocking dependencies
-chainbridge = { git = "https://github.com/centrifuge/chainbridge-substrate.git", default-features = false, branch = "master" }
-node-primitives = { git = "https://github.com/centrifuge/substrate",  default-features = false, branch = "master" }
+#chainbridge = { git = "https://github.com/centrifuge/chainbridge-substrate.git", default-features = false, branch = "master" }
+chainbridge = { git = "https://github.com/tankofzion/chainbridge-substrate.git", default-features = false, branch = "refactor/chainbridge_pallet" }
```

Comment:
youcan remove this if there is already a direct dependency of chain bridge above